ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__cr-4.19-nightly.yaml

```diff
@@ -24,7 +24,7 @@ resources:
       memory: 200Mi
 tests:
 - as: krkn-hub-aws
-  cron: 0 6 * * 2,5
+  cron: 0 0 * * *
```

Comment:
lets change the time of these that the 3 jobs aren't all running at the same time of day

Reply:
Done!